lib/order.rb

```diff
@@ -0,0 +1,9 @@
+class Order
+  attr_accessor :id, :customer_name, :date, :item_id
+
+  def initialize(customer_name, date, item_id)
+    @customer_name = customer_name
+    @date = date
+    @item_id = item_id
+  end
```

Comment:
I like the combination of these two as you can set up an object much quicker using `order = Order.new(customer_name, date, item_id) ` as opposed to `order = Order.new \n order.customer_name = "FooBar"` and get + set the attributes from outside the class when you want to make changes to them